tsl/test/expected/partialize_finalize.out

```diff
@@ -142,42 +142,68 @@ select a, _timescaledb_internal.finalize_agg( 'sum(double precision)', null, nul
- a  | sumb | minc  |             maxd             
-----+------+-------+------------------------------
-  1 |  150 | abc   | Sun Jan 03 09:00:00 2010 PST
-  2 |   60 | hello | Fri Jan 01 09:00:00 2010 PST
-  3 |   90 |       | 
- 11 |      | hello | Sat Jan 02 06:00:00 2010 PST
- 12 |   10 | hello | Sat Jan 02 06:00:00 2010 PST
+, _timescaledb_internal.finalize_agg( 'max(timestamp with time zone)', null, null, null, partiald, null::timestamptz ) maxd
+, _timescaledb_internal.finalize_agg( 'stddev(numeric)', null, null, null, partiale, null::numeric ) stddevb
+, _timescaledb_internal.finalize_agg( 'stddev(int8)', null, null, null, partialf, null::numeric ) stddeve
+from t1 group by a order by a ;
+ a  | sumb | minc  |             maxd             |       stddevb       |       stddeve       
+----+------+-------+------------------------------+---------------------+---------------------
+  1 |  150 | abc   | Sun Jan 03 09:00:00 2010 PST | 15.8113883008418967 | 15.8113883008418967
+  2 |   60 | hello | Fri Jan 01 09:00:00 2010 PST | 10.0000000000000000 | 10.0000000000000000
+  3 |   90 |       |                              |  7.0710678118654752 |  7.0710678118654752
+ 11 |      | hello | Sat Jan 02 06:00:00 2010 PST |                     |                    
+ 12 |   10 | hello | Sat Jan 02 06:00:00 2010 PST |                     |                    
+(5 rows)
+
+-- overwrite partials with binary values from PostrgeSQL 13 and older --
+TRUNCATE TABLE t1;
+\COPY t1 FROM partialize_finalize_data.csv WITH CSV HEADER
+--repeat query to verify partial serialization sanitization works for versions PG >= 14
+select a, _timescaledb_internal.finalize_agg( 'sum(numeric)', null, null, null, partialb, null::numeric ) sumb
+, _timescaledb_internal.finalize_agg( 'min(text)', 'pg_catalog', 'default', null, partialc, null::text ) minc
+, _timescaledb_internal.finalize_agg( 'max(timestamp with time zone)', null, null, null, partiald, null::timestamptz ) maxd
+, _timescaledb_internal.finalize_agg( 'stddev(numeric)', null, null, null, partiale, null::numeric ) stddevb
+, _timescaledb_internal.finalize_agg( 'stddev(int8)', null, null, null, partialf, null::numeric ) stddeve
+from t1 group by a order by a ;
+ a  | sumb | minc  |             maxd             |       stddevb       |       stddeve       
+----+------+-------+------------------------------+---------------------+---------------------
+  1 |  150 | abc   | Sun Jan 03 09:00:00 2010 PST | 15.8113883008418967 | 15.8113883008418967
+  2 |   60 | hello | Fri Jan 01 09:00:00 2010 PST | 10.0000000000000000 | 10.0000000000000000
+  3 |   90 |       |                              |  7.0710678118654752 |  7.0710678118654752
+ 11 |      | hello | Sat Jan 02 06:00:00 2010 PST |                     |                    
+ 12 |   10 | hello | Sat Jan 02 06:00:00 2010 PST |                     |                    
```

Comment:
I'm not saying to replace the current queries to this... just add it at the end as an extra step

Reply:
I just tested this and it prints the same output thrice, once for t1, once for t2, and once for the union. I am not sure it's very useful @fabriziomello

Reply:
Ooops... my bad, actually I meant to use `EXCEPT` instead of `UNION`... but anyway returning the same result as before means that ALL rows are equal in both tables.

What I meant was something like:

```sql
fabrizio=# (select a, _timescaledb_internal.finalize_agg( 'sum(numeric)', null, null, null, partialb, null::numeric ) sumb
, _timescaledb_internal.finalize_agg( 'min(text)', 'pg_catalog', 'default', null, partialc, null::text ) minc
, _timescaledb_internal.finalize_agg( 'max(timestamp with time zone)', null, null, null, partiald, null::timestamptz ) maxd
, _timescaledb_internal.finalize_agg( 'stddev(numeric)', null, null, null, partiale, null::numeric ) stddevb
, _timescaledb_internal.finalize_agg( 'stddev(int8)', null, null, null, partialf, null::numeric ) stddeve
from t1 group by a order by a)
EXCEPT
(select a, _timescaledb_internal.finalize_agg( 'sum(numeric)', null, null, null, partialb, null::numeric ) sumb
, _timescaledb_internal.finalize_agg( 'min(text)', 'pg_catalog', 'default', null, partialc, null::text ) minc
, _timescaledb_internal.finalize_agg( 'max(timestamp with time zone)', null, null, null, partiald, null::timestamptz ) maxd
, _timescaledb_internal.finalize_agg( 'stddev(numeric)', null, null, null, partiale, null::numeric ) stddevb
, _timescaledb_internal.finalize_agg( 'stddev(int8)', null, null, null, partialf, null::numeric ) stddeve
from t2 group by a order by a);
 a | sumb | minc | maxd | stddevb | stddeve 
---+------+------+------+---------+---------
(0 rows)
```